### PR TITLE
Replaced line: "It's pretty simple...  "

### PR DESCRIPTION
**Pull request by KirstyPringle**
Replaced Line:

It's pretty simple:
every alphanumeric character matches against itself.
After the pattern comes the name or names of the files we're searching in.

With:

The grep command searches through the file, looking for matches to the pattern specified.

To use it type grep, then the pattern we're searching for and finally the name of the file (or files) we're searching in.

(As I think it reads more clearly).

### CODE REVIEW

File: _episodes/07-find.md [hunk 1]
```diff
@@ -75,9 +75,11 @@ Today it is not working
 {: .output}
 
 Here, `not` is the pattern we're searching for.
-It's pretty simple:
-every alphanumeric character matches against itself.
-After the pattern comes the name or names of the files we're searching in.
+
```
Comment (rgaiacs) [inline _episodes/07-find.md]: Remove this line.

File: _episodes/07-find.md [hunk 2]
```diff
@@ -75,9 +75,11 @@ Today it is not working
...
+The grep command searches through the file, looking for matches to the pattern specified.
+
```
Comment (rgaiacs) [inline _episodes/07-find.md]: Remove this line.

**Comment by KirstyPringle** [comment]
Hi,  that's fine.  I have implemented the changes you suggest.

**Comment by gdevenyi** [comment]
Thanks!

**Pull request merged**